README.md

```diff
@@ -55,7 +55,9 @@ cp .env.example .env
 Install dependencies
 
 ```
-pip3 install -r requirements.txt
+python3 -m venv .env
+source .env/bin/activate 
+pip install -r requirements.txt
 ```
 
```

Comment:
pip3 is needed, right ??
better to use both `virtualenv` and `virtualenvwrapper`

Reply:
pipenv solves the issue.

Reply:
Since venv was created with `python3`, `pip` inside `venv` would suffice.

Reply:
> pipenv solves the issue.

Setting up `pipenv` on different OSes has multiple issues, It's better to stick to basics for quick setup.

Reply:
better to include a pip update option before installing requirements,  without this pip results in error in some OSes. 
`pip install --upgrade pip`

Reply:
There were some migration issue on `pip 9` to `pip 10`, It's better to leave `pip` version management to the user.